src/containers/InfographicContainer.jsx

```diff
@@ -367,25 +365,14 @@ class InfographicContainer extends React.Component {
     const infographicKeys: Array<string> = Object.keys(this.state.infographics)
 
     return (
-      <div className='flex-box'>
-        <SideBar
-          menu={{
-            // for pulling out explorer buttons
-            data: infographicKeys,
-            // update which infographic we're looking at
-            handler: this._sidebarToggle.bind(this),
-            // the current active infographic
-            selected: this.state.selected,
-            // sidebar header
-            title: 'Explore the data',
-          }}
-        />
-        <section className='float-r infographic-container'>
+        <section className='float-r infographic-container' style={{paddingTop: 25}}>
```

Comment:
Where possible, try to eliminate inline styling. If we can add it to the infographic-container class, that'd be ideal. If not, no worries.